acceptance-tests/src/test/java/tech/pegasys/ethsigner/tests/UpCheckAcceptanceTest.java

```diff
@@ -15,30 +15,51 @@
 import static org.assertj.core.api.Assertions.assertThat;
 
 import tech.pegasys.ethsigner.tests.dsl.http.HttpResponse;
+import tech.pegasys.ethsigner.tests.dsl.node.NodeConfiguration;
+import tech.pegasys.ethsigner.tests.dsl.node.NodeConfigurationBuilder;
+import tech.pegasys.ethsigner.tests.dsl.node.NodePorts;
+import tech.pegasys.ethsigner.tests.dsl.signer.Signer;
+import tech.pegasys.ethsigner.tests.dsl.signer.SignerConfiguration;
+import tech.pegasys.ethsigner.tests.dsl.signer.SignerConfigurationBuilder;
 
 import java.net.SocketTimeoutException;
 
 import io.netty.handler.codec.http.HttpResponseStatus;
+import org.junit.jupiter.api.BeforeAll;
 import org.junit.jupiter.api.Test;
 
-public class UpCheckAcceptanceTest extends AcceptanceTestBase {
+public class UpCheckAcceptanceTest {
 
   private static final String UP_CHECK_PATH = "/upcheck";
   private static final String UP_CHECK_MESSAGE = "I'm up!";
-  private static final String TIMEOUT_MESSAGE = "timeout";
+
+  private static Signer ethSigner;
+
+  @BeforeAll
+  public static void setUpBase() {
+    Runtime.getRuntime()
+        .addShutdownHook(new Thread(DataPathFeatureFlagAcceptanceTest::tearDownBase));
```

Comment:
This shutdown hook will use the static fields from DataPathFeatureFlagAcceptanceTest not from your test and so will not clean up properly.